.github/workflows/update-model-info.yml

```diff
@@ -1,28 +0,0 @@
-name: update-model-info
```

Comment:
maybe we should keep this for some time in case there's some issue during the transition from github to ngc

Reply:
sounds good @yiheng-wang-nv then in the future we can easily deprecate the github host by removing the files.

Reply:
Good idea @wyli , maybe we can also discuss about the expected changes in MONAI core (monai.bundle.scripts), let me open a new ticket.